cherab/tools/observers/bolometry.py

```diff
@@ -687,6 +695,350 @@ def etendue_single_run(_):
         return etendue, etendue_error
 
 
+class BolometerIRVB(TargettedCCDArray):
+    """
+    A rectangular infra red video bolometer (IRVB).
+
+    Can be configured to sample a single ray per pixel, or fan of rays
+    oriented along the observer's z axis.
+
+    :param str detector_id: The name for this detector.
```

Comment:
What was wrong with `name` as defined for a node?

Reply:
This breaks consistency with the rest of the underlying framework. This ends up as 'name' anyway... so just use `name`.

Reply:
Done, though `BolometerFoil` still uses `detector_id` for now.